test/fixtures/v5/arrive_waypoint/left.json

```diff
@@ -0,0 +1,25 @@
+{
+    "step": {
+        "maneuver": {
+            "type": "arrive",
+            "modifier": "left"
+        },
+        "name": "Street Name"
+    },
+    "instructions": {
+        "de": "Sie haben Ihr erste Ziel erreicht, es befindet sich links von Ihnen",
```

Comment:
We'll need gendering of the adjective here 😢 Maybe we can get around this with some smart same-gender naming, I'll have a look ...

Now: `Sie haben Ihr erste Ziel erreicht`.
Correct: `Sie haben Ihr erstes Ziel erreicht`.